README.md

```diff
@@ -747,6 +748,38 @@ This is how it can be included in your `configuration.nix`:
 }
 ```
 
+## Use with home manager
+
+sops-nix also provides a home-manager module.
+This module provides a subset of features provided by the system-wide sops-nix since features like the creation of the ramfs and changing the owner of the secrets are not available for non-root users.
+
+Instead of running as an activation script, sops-nix runs as a systemd user service called `sops-nix.service`.
+And instead of decrypting to `/run/secrets`, the secrets are decrypted to `$XDG_RUNTIME_DIR/secrets`.
```

Comment:
`$XDG_RUNTIME_DIR` can be not set on non-nixos. We should recommend to have it point it to a tmpfs so that secrets are not persisted stored in plain text or otherwise live with the consequences...